pkg/services/opsgenie/opsgenie_config.go

```diff
@@ -0,0 +1,37 @@
+package opsgenie
+
+import (
+	"net/url"
+
+	"github.com/containrrr/shoutrrr/pkg/services/standard"
+)
+
+// Config for use within the opsgenie plugin
+type Config struct {
+	ApiKey string `desc:"The OpsGenie API key"`
+	Host   string `desc:"The OpsGenie API host. Use 'api.opsgenie.com' for US and 'api.eu.opsgenie.com' for EU instances"`
+	standard.QuerylessConfig
```

Comment:
The params on send should probably (at least some of them) be settable using the query string.